pom.xml

```diff
@@ -9,7 +9,7 @@
 
   <groupId>net.jodah</groupId>
   <artifactId>lyra</artifactId>
-  <version>0.5.4-SNAPSHOT</version>
+  <version>0.5.5-SNAPSHOT</version>
```

Comment:
Never change versions in pull requests. It's up to the maintainers to decide when to do that (and what the new version should be).